frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -56,10 +56,28 @@
         order.update_column(:state, "address")
       end
 
-      it "should associate the order with a user" do
-        order.update_column :user_id, nil
-        expect(order).to receive(:associate_user!).with(user)
-        get :edit
+      context "current behavior" do
+        before do
+          stub_spree_preferences(associate_user_in_authentication_extension: false)
+        end
+
+        it "should associate the order with a user" do
+          order.update_column :user_id, nil
+          expect(order).to receive(:associate_user!).with(user)
+          get :edit
+        end
+      end
+
+      context "new behavior" do
```

Comment:
Similar to the above comment, this may be better as `"with associate_user_in_authentication_extension disabled"`.